python/ray/train/torch/config.py

```diff
@@ -8,11 +8,16 @@
 import torch.distributed as dist
 
 import ray
+from ray._private.accelerators.hpu import HPU_PACKAGE_AVAILABLE
 from ray.train._internal.utils import get_address_and_port
 from ray.train._internal.worker_group import WorkerGroup
 from ray.train.backend import Backend, BackendConfig
 from ray.util import PublicAPI
 
+if HPU_PACKAGE_AVAILABLE:
+    import habana_frameworks.torch.core as htcore  # noqa: F401
+    import habana_frameworks.torch.distributed.hccl as hpu_dist  # noqa: F401
```

Comment:
Should this import logic be moved to within `_setup_torch_process_group` and only called when `backend == "hccl"`? Seems like this is needed to be called on the workers.

Reply:
updated

Reply:
@kira-lin can you push the change to the PR?

Reply:
sorry, forgot 🥲 pushed